### ISSUE TITLE
Foundation 6?

### ISSUE THREAD

**Issue by GoTeamScotch** [milohuang/reverie#326]
Foundation 6 has been out for some time and Reverie is based on 5. Version 6 offers some great new features (particularly flex grids). Are there any plans to upgrade to Foundation 6? Is this project even maintained any longer?

**Comment by jessedmatlock** [comment]
+1

**Comment by gpspake** [comment]
This project hasn't been maintained for nearly 3 years, now. I maintained a fork for a while but abandoned it when 6 was released. I would recommend [FoundationPress](https://github.com/olefredrik/FoundationPress) as it's essentially become the de-facto Foundation WordPress theme and it's actively maintained. I've been building a theme on top of it and can attest that it's well organized and the author has responded to one or two of my PR's quickly.
